openmoltools/tests/test_openeye.py

```diff
@@ -11,7 +11,7 @@
 import openmoltools.openeye
 import pandas as pd
 import mdtraj as md
-from mdtraj.testing import raises
+from numpy.testing import assert_raises
```

Comment:
Just for my curiosity, are there advantages of using `numpy.testing.assert_raises` over `nose.tools.assert_raises`?

Reply:
The `nose` test is the correct way to go, but this minimizes package drift, which I was trying to do with this PR. `mdtraj` no longer has `raises`, but now just passes through `numpy.testing.assert_raises` as its own, so I just pulled that in to avoid having people with mdtraj get an error based on version.

In some larger PR/cleanup, I fully think this should be replaced with `nose`'s `assert_raises` function.